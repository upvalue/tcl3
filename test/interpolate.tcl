set x 2
set y 2
puts "$x + $y = 4"
