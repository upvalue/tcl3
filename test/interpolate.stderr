{"type": "TK_ESC", "begin": 0, "end": 3, "body": "set"}
{"type": "TK_SEP", "begin": 3, "end": 4, "body": " "}
{"type": "TK_ESC", "begin": 4, "end": 5, "body": "x"}
{"type": "TK_SEP", "begin": 5, "end": 6, "body": " "}
{"type": "TK_ESC", "begin": 6, "end": 7, "body": "2"}
{"type": "TK_EOL", "begin": 7, "end": 8, "body": "\n"}
{"type": "TK_ESC", "begin": 8, "end": 11, "body": "set"}
{"type": "TK_SEP", "begin": 11, "end": 12, "body": " "}
{"type": "TK_ESC", "begin": 12, "end": 13, "body": "y"}
{"type": "TK_SEP", "begin": 13, "end": 14, "body": " "}
{"type": "TK_ESC", "begin": 14, "end": 15, "body": "2"}
{"type": "TK_EOL", "begin": 15, "end": 16, "body": "\n"}
{"type": "TK_ESC", "begin": 16, "end": 20, "body": "puts"}
{"type": "TK_SEP", "begin": 20, "end": 21, "body": " "}
{"type": "TK_ESC", "begin": 22, "end": 22, "body": ""}
{"type": "TK_VAR", "begin": 23, "end": 24, "body": "x"}
{"type": "TK_ESC", "begin": 24, "end": 27, "body": " + "}
{"type": "TK_VAR", "begin": 28, "end": 29, "body": "y"}
{"type": "TK_ESC", "begin": 29, "end": 33, "body": " = 4"}
{"type": "TK_EOL", "begin": 34, "end": 35, "body": "\n"}
{"type": "TK_EOF", "begin": 34, "end": 35, "body": "\n"}
